src/app/story/storyrow/totaltimes/totaltimes.component.css

```diff
@@ -0,0 +1,27 @@
+.timeline-wrapper {
+    display: none;
+}
+
+@media (min-width: 992px) {
+    .timeline-wrapper {
+        display: initial;
+    }
+}
+
+.timeline-data {
```

Comment:
Could you rename this to match the BEM convention? Something like `timeline-properties__row__data`. This should also be at the end of the CSS file.